ORM Cookbook/Recipes.ServiceStack/Entities/Department.cs

```diff
@@ -0,0 +1,19 @@
+﻿using ServiceStack.DataAnnotations;
+
+namespace Recipes.ServiceStack.Entities
+{
+    [Schema("HR")]
+    [Alias("Department")]
```

Comment:
Sigh, I really wish ORMs would just use the `Table` attribute instead of making up their own stuff.

Reply:
ServiceStack created it's annotations API back in 2011 (https://github.com/ServiceStack/ServiceStack/blob/613c4c09ca7160caf66b48ab4baedf6e219d602e/src/ServiceStack.Interfaces/DataAnnotations/AliasAttribute.cs). I've been able to refactor ServiceStack codebases for half a decade with very minimal changes, even when .NET core came around. They have good reasons for maintaining their own set of conventions.

Reply:
That's fair for ServiceStack, but there's no excuse for ORM created since .NET 4.5.